### PR TITLE
MSTest nuget packages upgrade

### PR DESCRIPTION
**Pull request by AbhitejJohn**
Updating the MSTest nuget packages to the ones shipping with VS, desktop and UWP based projects.

### CODE REVIEW

**Comment by AbhitejJohn** [comment]
@srivatsn to review.

File: build.ps1 [hunk 1]
```diff
@@ -45,7 +45,7 @@ if ($Verbosity -eq 'diagnostic') {
 }
 
 # Install a stage 0
-$DOTNET_INSTALL_SCRIPT_URL="https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/scripts/obtain/dotnet-install.ps1"
+$DOTNET_INSTALL_SCRIPT_URL="https://raw.githubusercontent.com/dotnet/cli/feature/msbuild/scripts/obtain/dotnet-install.ps1"
```
Comment (srivatsn) [inline build.ps1]: You seem to fixed merge conflicts incorrectly. This change and the netci.groovy change shouldn't be there.

**Review by srivatsn — CHANGES_REQUESTED** [review]

**Review by srivatsn — APPROVED** [review]

**Comment by srivatsn** [comment]
Tagging @MattGertz. This is to sync up to a MSTest versions already inserted to VS.

**Comment by natidea** [comment]
Approved to merge via [374631](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/374631)

**Pull request merged**